x-pack/plugins/lists/server/services/utils/escape_query.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const escapeQuotes = (str: string): string => {
+  return str.replace(/[\\"]/g, '\\$&');
+};
```

Comment:
Is this the same as this one?
x-pack/plugins/data_enhanced/public/autocomplete/providers/kql_query_suggestion/lib/escape_kuery.ts

That looks like it only works client side and not server and isn't part of a common section for usage in both areas which is a bummer.

Wish it was in one common spot. I'm fine with this duplicated here though. I would just copy over their tests and put a note that there is a duplication between the two in case someone sees them deviate later/change.

Reply:
Yep it's the same function, definitely would like to have it in a common place along with the other escaping utilities (and maybe a more robust parameterization scheme like Ross suggested above). For now though it seems a simple enough function to just do the ol' copy paste.